### PR TITLE
fixes #8581 - [vmware] use cluster full path to create vm

### PR DESCRIPTION
**Pull request by shlomizadok**
**To be merged only when fog/fog#3341 is merged and new fog version is out.**

### CODE REVIEW

File: app/models/compute_resources/foreman/model/vmware.rb [hunk 1]
```diff
@@ -50,7 +50,10 @@ def cluster(cluster)
     end
 
     def clusters
-      name_sort(dc.clusters)
+      clusters = dc.clusters.each do |cluster|
```
Comment (ohadlevy) [inline app/models/compute_resources/foreman/model/vmware.rb]: nitpick1: 

``` ruby
name_sort(dc.clusters.map(&:name_path))
```

nitpick2: name_path sounds odd, maybe full_path? or something similar?

Comment (shlomizadok) [inline app/models/compute_resources/foreman/model/vmware.rb]: renamed to pull_path. name_sort expects an array with objects with name. and I also want to ensure that name == full_path

**Comment by ohadlevy** [comment]
fog/fog#3341 is merged, pending new fog release

**Comment by shlomizadok** [comment]
blocked by #2095.

**Comment by shlomizadok** [comment]
@domcleal -- can you please merge this pr, now that #2095 is merged? thanks

**Comment by domcleal** [comment]
I won't merge it, I haven't reviewed nor am I able to test it.

**Comment by shlomizadok** [comment]
fair enough. thanks.

**Comment by ohadlevy** [comment]
merged as 83e22d2 thanks @shlomizadok

**Pull request closed without merging**